test/mocks/handlers/tripReq.handler.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/prefer-default-export */
+import { rest } from 'msw';
+import baseURl from '../../../src/api/dummy';
+
+export const tripReqHandlers = [
+  rest.get('https://mocki.io/v1/0a16d481-3752-4cc6-b7f8-ca40f1b95a3d', (req, res, ctx) => {
+    if (req.body.password.length < 8) {
+      return res(
+        ctx.status(400),
+        ctx.json({
+          Error:
+            'Password:eight characters, at least one letter, one number and one special character',
+          status: 400,
+        }),
+        ctx.delay(100)
+      );
+    }
+    return res(ctx.status(201), ctx.json({ Message: 'User created', status: 201 }), ctx.delay(100));
```

Comment:
Add the mocks results for the endpoint, maybe the trip json object you expect from the server. Otherwise if you make a request to that endpoint it will try to find `req.body.password` when theres'n one to begin with.